src/main/java/nwoolcan/viewmodel/warehouse/QueriedStockViewModel.java

```diff
@@ -0,0 +1,30 @@
+package nwoolcan.viewmodel.warehouse;
+
+import nwoolcan.viewmodel.warehouse.stock.StockViewModel;
+
+import java.util.List;
+
+/**
+ * Model of the view of the list of the stocks.
+ */
+public final class QueriedStockViewModel {
```

Comment:
i dont see the use case of this view model.
if you just need to display one single thing, (a list) just use that